File: 2018/J1.py
q = input()
w = input()
e = input()
r = input()

an = True
if q == "8" or q == "9":
    if r == "8" or r == "9":
        if w == e:
            print("ignore")
            an = False

if an:
    print("answer")

File: 2018/J1.cpp
#define _CRT_SECURE_NO_WARNINGS 1
#include <fstream>


using namespace std;

int main() {
	int first, second, third, fourth;
	bool ignore = false;
	scanf("%d %d %d %d", &first, &second, &third, &fourth);
	if (first == 9 || first == 8)
	{
		if(fourth == 9 || fourth == 8)
		{
			if (second == third)
			{
				printf("ignore");
				ignore = true;
			}
		}
	}
	if (!ignore)
	{
		printf("answer");
	}
	return 0;
}